sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -132,6 +132,19 @@ object StaticSQLConf {
       .intConf
       .createWithDefault(1000)
 
+  val BROADCAST_EXCHANGE_MAX_THREAD_THREASHOLD =
```

Comment:
nit: `BROADCAST_EXCHANGE_MAX_THREAD_THREASHOLD` -> `BROADCAST_EXCHANGE_MAX_THREAD_THRESHOLD`

Reply:
oh sorry for this error,i have fixed that